homeassistant/components/shelly/strings.json

```
@@ -31,7 +32,8 @@
       "cannot_connect": "[%key:common::config_flow::error::cannot_connect%]",
       "invalid_auth": "[%key:common::config_flow::error::invalid_auth%]",
       "unknown": "[%key:common::config_flow::error::unknown%]",
-      "firmware_not_fully_provisioned": "Device not fully provisioned. Please contact Shelly support"
+      "firmware_not_fully_provisioned": "Device not fully provisioned. Please contact Shelly support",
+      "custom_port_not_supported": "Gen1 devices cannot be configured via range extender."
```

PR description mentions devices behind NAT (which is correct), and the error is `custom_port_not_supported`, the description should be similar.
```suggestion
      "custom_port_not_supported": "Gen1 devices does not support custom port."
```

---

Done